src/raster/rasterband.rs

```diff
@@ -847,6 +848,96 @@ impl<'a> RasterBand<'a> {
             max: min_max[1],
         })
     }
+
+    /// Fetch default raster histogram.
+    ///
+    /// # Arguments
+    ///
+    /// * `force` - If `true`, force the computation. If `false` and no default histogram is available, the method will return None
+    pub fn get_default_histogram(&self, force: bool) -> Result<Option<Histogram>> {
```

Comment:
For coherence with other methods/functions, I'd suggest collapsing these two methods into a single one accepting a new `HistogramOptions` with the options, and implement `Default` for it.

Reply:
Fair point. Let's hear what some of the other contributors think. I'm on the extreme of "let's not leak C-language workarounds" and "let's leverage Rust to improve ergonomics". Others, I think, are more in your camp, so let's let someone else make the call. We've had related debates [here](https://github.com/georust/gdal/pull/456).